src/sctools/test/test_entrypoints.py

```diff
@@ -24,3 +24,29 @@ def test_Attach10XBarcodes_entrypoint():
             assert isinstance(alignment.get_tag('SY'), str)
             assert isinstance(alignment.get_tag('SR'), str)
     os.remove('test_tagged_bam.bam')  # clean up
+
+
+def test_Attach10XBarcodes_entrypoint_with_whitelist():
+    args = [
+        '--r1', data_dir + 'test_r1.fastq',
+        '--i1', data_dir + 'test_i7.fastq',
+        '--u2', data_dir + 'test_r2.bam',
+        '--output-bamfile', 'test_tagged_bam.bam',
+        '--whitelist', data_dir + '1k-august-2016.txt']
+
+    rc = platform.TenXV2.attach_barcodes(args)
```

Comment:
Hopefully the last one here...but what is `rc`?

Reply:
Return Code. I'll be explicit.

Reply:
thanks!